README.md

```diff
@@ -425,6 +425,27 @@ Given any valid representation of an address return the normalized representatio
 '0xc305c901078781c232a2a521c2af7980f8385ee9',
 ```
 
+### Conversion Utils
+
+
+These methods convert values using standard practices in the Ethereum ecosystem.
+For example, strings are encoded to binary using UTF-8.
+
+See the
+[web3.py docs](http://web3py.readthedocs.io/en/stable/overview.html#overview-type-conversions)
```

Comment:
That section of the web3 docs should probably be ported or duplicated here.  Feels weird to have a circular dependency of sorts and I like `eth-utils` standing on it's own with no need to go elsewhere for documentation.

Also would request a block of examples for each of these as I think it's the most expressive way to demonstrate the functionality.